### PR TITLE
feat: Add installing node dependencies with Gemfury

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: node/install-dependencies-with-gemfury/action.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Install Dependencies with Gemfury
+description: Installs dependencies with Triax's Gemfury registry
+
+runs:
+  using: composite
+  steps:
+    - name: Authenticate Gemfury
+      run: echo "//npm-proxy.fury.io/triaxtec/:_authToken=$GEMFURY_PULL_TOKEN" > .npmrc
+      env:
+        GEMFURY_PULL_TOKEN: ${{ secrets.GEMFURY_PULL_TOKEN }}
```
Comment (dbanty) [inline node/install-dependencies-with-gemfury/action.yml]: Can you use secrets cross-action like this? I thought you had to provide them as an input

**Review by dbanty — COMMENTED** [review]
Just one question before approval.

File: node/install-dependencies-with-gemfury/action.yml (hunk 1 above)
Comment (ghost) [reply node/install-dependencies-with-gemfury/action.yml]: Good catch - you're correct

Comment (dbanty) [reply node/install-dependencies-with-gemfury/action.yml]: You think it's worth also making the endpoint configurable? At some point we're going to move our private dependencies to GitHub Packages I think.

Comment (ghost) [reply node/install-dependencies-with-gemfury/action.yml]: I think that's worth doing.  I considered it before making this PR, but didn't think there was a high chance would change registries any time soon.  It sounds like that was a poor assumption on my part.

I'll add that as an input.

Comment (ghost) [reply node/install-dependencies-with-gemfury/action.yml]: Alright, this is now registry agnostic and using inputs.

**Review by dbanty — APPROVED** [review]

**Comment by dbanty** [comment]
Awesome, thanks! Give it a try in prox-trace-ui and let me know how it goes

**Pull request merged**